stdlib/public/Darwin/Foundation/Foundation.swift

```diff
@@ -29,7 +29,153 @@ public let NSNotFound: Int = .max
 // NSLocalizedString
 //===----------------------------------------------------------------------===//
 
-/// Returns a localized string, using the main bundle if one is not specified.
+/// Returns the localized version of a string.
+///
+/// - parameter key: An identifying value used to reference a localized string.
+///   `key` must not be the empty string. Values keyed by the empty string will
+///   not be localized.
```

Comment:
Okay, I see — we use "must not" to mean preconditions, which isn't exactly the behavior here. I think you could change this to something like "Don't pass an empty string as a key. Values keyed by an empty string are not localized."